src/algebra/category/Group/colimits.lean

```diff
@@ -300,7 +300,7 @@ open quotient_add_group
 The categorical cokernel of a morphism in `AddCommGroup`
 agrees with the usual group-theoretical quotient.
 -/
-noncomputable def cokernel_iso_quotient {G H : AddCommGroup} (f : G ⟶ H) :
+noncomputable def cokernel_iso_quotient {G H : AddCommGroup.{u}} (f : G ⟶ H) :
```

Comment:
Are `G` and `H` not allowed to reside in different universes?

Reply:
no, `G ⟶ H` is only defined then `G` and `H` are in the same category, so in particular they must have the same type (including universe level)